integrations/Matomo/util.js

```diff
@@ -10,14 +11,14 @@ import { NAME } from "./constants";
 const goalIdMapping = (event, goalListMap, message) => {
   let revenue;
   if (message.properties) revenue = message.properties.revenue;
```

Comment:
`if (message.properties) revenue = message.properties?.revenue;`
change this to
`revenue = message.properties?.revenue`

Reply:
Done